### PR TITLE
Introducing qtspell

### PR DESCRIPTION
**Pull request by gbeine**
Spell checking for Qt text widgets

### CODE REVIEW

File: Library/Formula/qtspell.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'formula'
```
Comment (mistydemeo) [inline Library/Formula/qtspell.rb]: You can leave this line out.

File: Library/Formula/qtspell.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+class Qtspell < Formula
+  homepage 'https://github.com/manisandro/qtspell'
```
Comment (mistydemeo) [inline Library/Formula/qtspell.rb]: Double quotes everywhere, please!

File: Library/Formula/qtspell.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on 'enchant'
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
+  end
+
```
Comment (mistydemeo) [inline Library/Formula/qtspell.rb]: Leave this line out.

**Comment by mistydemeo** [comment]
Is it possible to write a test for this software?

**Comment by gbeine** [comment]
There is only a library installed, so I don't see how to test this.

**Comment by bfontaine** [comment]
You could write a small program that uses the library, compile and execute it, like what [`tinyxml2`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/tinyxml2.rb#L23-L33) does.

**Comment by gbeine** [comment]
This is a library offering spell checking for Qt text boxes.
The program would not be as small as the one for tinyxml2 is.
And it would force me to program in C++. At least this I'd like to avoid like hell.

**Comment by bfontaine** [comment]
There’s a small(ish) example in the upstream repo but it opens a window, which is not what we want.

> And it would force me to program in C++. At least this I'd like to avoid like hell.

I understand that :wink: 

@mistydemeo Would a test where we compile a program but don’t run it be ok here?

**Comment by mikemcquaid** [comment]
> This is a library offering spell checking for Qt text boxes.
> The program would not be as small as the one for tinyxml2 is.
> And it would force me to program in C++. At least this I'd like to avoid like hell.

Please do so, thanks. Out of interest: why do you want to add this library if you aren't going to write Qt/C++?

**Comment by gbeine** [comment]
I'd like to add a formula for a GUI for tesseract that depends on qtspell.
An I have no C++ programming skills ;-)
Just Perl & Java... and maybe PHP if you accept it as programming language.

**Comment by bfontaine** [comment]
Any update on this?

**Comment by DomT4** [comment]
Closing due to no response. Will welcome a new PR with the issues highlighted addressed. Thanks for your submission to Homebrew!

**Pull request closed without merging**